src/Generator.cpp

```diff
@@ -1164,14 +1381,6 @@ GeneratorParamBase::~GeneratorParamBase() {
 }
 
 void GeneratorParamBase::check_value_readable() const {
```

Comment:
Why did the assert go away?

Reply:
...I don't remember. Let me figure it out.